portal-web/test/functional/com/liferay/portalweb/paths/portlet/marketplace/AppManager.path

```diff
@@ -11,7 +11,7 @@
 <!--APP-->
 <tr>
 	<td>APP_NAME</td>
-	<td>//div[contains(@class,'content')]//li[contains(@class,'list-group-item')]//a[contains(.,'${key_appName}')]</td>
+	<td>//div[contains(@class,'content')]//li[contains(@class,'list-group-item')]//*[contains(.,'${key_appName}')]</td>
```

Comment:
fyi @patriciajeanperez, I'll be making this change here. May want to consider refactoring the other similar AppManager paths too

Reply:
thanks, @john-co! fyi, @sharryxue